docs/source/deploying/simple.md

```diff
@@ -65,6 +65,7 @@ location ~ / {
   add_header Cache-Control "public";
   expires +1m;
   proxy_pass http://volto;
+  }
```

Comment:
Please use consistent 4-space indentation throughout this example. [nginx examples use 4 spaces for indentation](https://docs.nginx.com/nginx/admin-guide/web-server/web-server/#virtual-server), and so should we.

Reply:
Done! @stevepiercy